public/css/styles.css

```diff
@@ -65,3 +87,106 @@ canvas {
 .submenu-option.selected {
   color: var(--theme-color-3);
 }
+
+#desktop {
+  background: linear-gradient(
+    #C09DF7,
+    var(--theme-color-5)
+  );
+  height: 100vh;
+  font-family: 'Trebuchet MS', sans-serif;
+}
+
+.desktop-button {
+  box-sizing: border-box;
+  border: none;
+  background: var(--surface);
+  box-shadow: var(--border-raised-outer), var(--border-raised-inner);
+  min-width: 260px;
+  min-height: 30px;
+  margin-left: 5px;
+  font-family: 'Trebuchet MS', sans-serif;
+  font-size: 10pt;
+}
+
+.desktop-button:active {
+  box-sizing: border-box;
+  outline: 1pt;
+  outline-style: dotted;
+  outline-offset: -5px;
+  background: var(--surface);
+  box-shadow: var(--border-sunken-outer), var(--border-sunken-inner);
+  min-width: 260px;
+  min-height: 30px;
+  margin-left: 5px;
+  font-family: 'Trebuchet MS', sans-serif;
+  font-size: 10pt;
+}
+
+input[type=text], select {
+  width: 260px;
+  padding: 6px;
+  margin-top: 5px;
+  display: inline-block;
+  border-width: var(--border-width);
+  box-sizing: border-box;
+  box-shadow: var(--border-sunken-outer), var(--border-sunken-inner);
+}
+
+input[type=password], select {
+  width: 260px;
+  padding: 6px;
+  margin-top: 5px;
+  display: inline-block;
+  border-width: var(--border-width);
+  box-sizing: border-box;
+  box-shadow: var(--border-sunken-outer), var(--border-sunken-inner);
+}
```

Comment:
these two rules are the same, so u don't need to declare them twice, instead state all the "selectors" that the rule should be applied to in a comma separated list: `input[type=text], input[type=password], select`